### DISCUSSION TITLE
How loss of capacity is calculated ?

### DISCUSSION THREAD

**Discussion by Akila1993** [pybamm-team/PyBaMM#3916] [Q&A]
Hello Everyone, @brosaplanella @TomTranter 

I am using OKane degradation sub-models with the DFN model and want to familiarize myself with how the losses are calculated, such as the variables mentioned below. 

Can someone guide me to where I can see the source code or any reference for this in the PyBaMM repository? 

-  'Loss of lithium to SEI [mol]',
-  'Loss of capacity to SEI [A.h]',
-  'Total lithium lost to side reactions [mol]',
-  'Total capacity lost to side reactions [A.h]',
-  'Negative electrode capacity [A.h]',
-  'Loss of active material in negative electrode [%]',
-  'Loss of lithium to lithium plating [mol]',
-  'Loss of capacity to lithium plating [A.h]',
-  'Loss of lithium to SEI on cracks [mol]',
-  'Loss of capacity to SEI on cracks [A.h]',
-  'Positive electrode capacity [A.h]',

**Comment by valentinsulzer** [comment]
Mostly in https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/models/full_battery_models/lithium_ion/base_lithium_ion_model.py
